### PR TITLE
fix: show claiming status instantly

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
- Currently, claiming status takes a while to appear

### CODE REVIEW

File: packages/app/providers/claim-provider.tsx [hunk 1]
```diff
@@ -52,7 +52,10 @@ export function ClaimProvider({ children }: ClaimProviderProps) {
         transactionHash: response.transaction_hash,
       });
 
+      console.log("damn dude ", response.is_complete);
       if (response.is_complete) {
+        console.log("damn dude ", snackbar, snackbar?.show);
```
Comment (axeldelafosse) [inline packages/app/providers/claim-provider.tsx]: 🤣

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

**Review by alantoa — APPROVED** [review]
LGTM ✨

File: packages/app/providers/claim-provider.tsx (hunk 1 above)
Comment (intergalacticspacehighway) [reply packages/app/providers/claim-provider.tsx]: i regret making this repo public 😭😭😭

**Pull request merged**